src/components/card/Card.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import styles from './Card.module.css'
+
+const Card = ({course}) => {
+
+    const createCourse = (course) => {
```

Comment:
write the html code in the return function or use the component as tag not a function call